source/read-from-mongodb.txt

```diff
@@ -42,6 +42,18 @@ Overview
 
          .. include:: /scala/filters.txt
 
+.. important:: Change Stream Schema Inference
+
+   When the {+driver-short+} infers the schema of a data frame 
```

Comment:
Q: should we rename this source constant since the product is not a driver?

Reply:
love it